lib/proofer/applicant.rb

```diff
@@ -7,7 +7,8 @@ class Applicant
     attr_accessor :prev_address1, :prev_address2, :prev_city, :prev_state, :prev_zipcode
     attr_accessor :ssn, :dob, :phone, :email
     attr_accessor :drivers_license_state, :drivers_license_id, :passport_id, :military_id
-    attr_accessor :ccn, :mortgage, :home_equity_line, :auto_loan, :bank_acct, :bank_routing
+    attr_accessor :ccn, :mortgage, :home_equity_line, :auto_loan
+    attr_accessor :checking_acct, :savings_acct, :bank_routing
```

Comment:
as an API, would it be clearer to have us provide consistent routing + account numbers, and a separate field for savings/checking? might map better with our UI?

```
bank_routing_number
bank_account_number
bank_account_type: [:checking, :savings]
```

Reply:
Yeah, that's how I did it originally, but switched over to this b/c the implementation in the mock proofer got a little clunky. I'm still not sure though and can switch back 🤔

Reply:
I CMD+Zed all the changes I made to remove `bank_acct_type`. Lmk if you like that better.